### DISCUSSION TITLE
Any plans to add the ability to create filament profiles that will show up on the AMS?

### DISCUSSION THREAD

**Discussion by sparton175** [OrcaSlicer/OrcaSlicer#945] [Ideas]
I just started using Orca this week when I received my X1C Combo Monday. So far I've been loving the slicer and the printer! The only real feature that I've thought could be added out be a way in the GUI to create the filament profile JSONs that are sent to the printer for selection in the AMS, like how we can pick the added PolyMaker filament or the generic options.

**Comment by SoftFever** [comment]
Bambu hard code filament info in their firmware for AMS.
We can't add new profiles to their AMS unfortunately.

**Reply by kolorkil** [reply]
Is it still the case? I'm able to add custom filament to AMS manually, but this could/should be possible to do from Orca Slicer too.
![2024-03-13 113331](https://github.com/SoftFever/OrcaSlicer/assets/13376455/0273b208-d3fd-4a41-b5ea-acbedaad7491)

**Reply by Yevgeniy-Gonzha** [reply]
orca 1.9.1 knows how to do it!
why can't orca 2.0 do it?

**Reply by SoftFever** [reply]
I can't select custom filament for my P1P even in 1.9
Can you please provide a bit more info

**Reply by Yevgeniy-Gonzha** [reply]
I'll teach you (see screenshots).

![01](https://github.com/SoftFever/OrcaSlicer/assets/123834053/4b0957ca-d3f2-49b8-be8d-1edbdefa27e2)
![02](https://github.com/SoftFever/OrcaSlicer/assets/123834053/5be50df5-6712-4949-83fc-bbd6ef33e1df)
![03](https://github.com/SoftFever/OrcaSlicer/assets/123834053/22dee64b-634e-47d9-8037-0458e0740bfe)
![04](https://github.com/SoftFever/OrcaSlicer/assets/123834053/b71bb748-fd58-40bb-8a6e-c50953decf4d)
